Searching Algorithm Java/binary search.java

```diff
@@ -0,0 +1,34 @@
+Binary Search in Java
+
+//code
+
```

Comment:
This would make the code not compile. 
We would require a fully working code in oder to help others who want to learn from this repository. :)